### ISSUE TITLE
Stopped working after Skype Web updates

### ISSUE THREAD

**Issue by xvitaly** [EionRobb/skype4pidgin#282]
Stopped working today. The errors are:
 * failed getting Registration Token;
 * remote server closed connection.

If it connects, all my friends are offline. Messages also can't be sent or received.

Some errors from debug log:
```
(16:22:07) util: Redirecting to  https://bay-client-s.gateway.messenger.live.com/v1/users/ME/endpoints/
(16:22:07) skypeweb: Messages host has changed to bay-client-s.gateway.messenger.live.com
(16:22:07) nss: partial certificate chain
(16:22:07) g_log: skypeweb_string_get_chunk: assertion 'haystack && start && end' failed
(16:22:07) g_log: skypeweb_string_get_chunk: assertion 'haystack && start && end' failed
(16:22:07) g_log: skypeweb_string_get_chunk: assertion 'haystack && start && end' failed
(16:22:07) connection: Connection error on 0x37de010 (reason: 2 description: Failed getting Registration Token)
(16:22:07) account: Disconnecting account easycoding (0x21f9810)
(16:22:07) connection: Disconnecting connection 0x37de010
(16:22:07) GLib: g_source_remove: assertion 'tag > 0' failed
(16:22:07) GLib: g_source_remove: assertion 'tag > 0' failed
(16:22:07) GLib: g_source_remove: assertion 'tag > 0' failed
```

**Comment by xvitaly** [comment]
Connected today without errors. It's very strange.

**Comment by szr8** [comment]
I am getting same errors again this morning. Yesterday it had started working later on again, but today it's back to erroring out and when it does "connect" it shows all contacts as off-line.

**Comment by phpony** [comment]
All my contacts are also offline. But without any error messages.

**Comment by xvitaly** [comment]
"Failed getting Registration Token" error today.

**Comment by szr8** [comment]
I'm getting "Failed getting Registration Token" again today as well after it had been working for a couple days.

**Comment by toadfan321** [comment]
I'm having a similar issue using skypeweb with bitlbee. After adding the account and attempting to connect, it says conecting, authenticating, then fails with "Failed getting skype token" I am not using a live account, my login is just a regular username, i can provide debug logs if needed.

**Comment by EionRobb** [comment]
More debug logs are always welcomed :)

**Comment by toadfan321** [comment]
Scratch my last comment, after further investigation of the debug logs i found that i had just put in a wrong password, now its working :P

**Comment by EionRobb** [comment]
Most of these "Failed getting Registration Token" errors were from a temporary downtime that the server was having, others are from NSS issues solved by upgrading to 2.10.11.  Please open a new issue if you're having more Registration Token errors :)
